lib/partials.ts

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright 2017 resin.io
```

Comment:
Do these copyright headers need to be updated?

Reply:
Fixed in latest commit